frontend/src/components/ProjectRecommendationCard/index.jsx

```diff
@@ -19,15 +19,11 @@ const ProjectRecommendationCard = ({ name, commoncolabsnum}) => {
         </Title>
       </Row>
       <Tags>
-      <Tag color={theme.main.colors.first} style={{borderRadius: "8px", height: "16px", fontSize: "10px", lineHeight: "14px"}}>
-          Deep Learning
-        </Tag>
-        <Tag color={theme.main.colors.first} style={{borderRadius: "8px", height: "16px", fontSize: "10px", lineHeight: "14px"}}>
-          Quantum Computing
-        </Tag>
-        <Tag color={theme.main.colors.first} style={{borderRadius: "8px", height: "16px", fontSize: "10px", lineHeight: "14px"}}>
-          Blockchain
-        </Tag>
+        { tags !== undefined ? tags.map((t) => {
+          return <Tag color={theme.main.colors.first} style={{borderRadius: "8px", height: "16px", fontSize: "10px", lineHeight: "14px"}}>
+            {t}
+          </Tag>
+        }): ":S"}
```

Comment:
but you need to give a key to the Tag component in that case since you are rendering multiple tag components consecutively, react needs a key prop for those to choose if one of them needs to be rerendered it rerenders it by looking at its key. If you don't give them a key prop react tries to render the each tag component where only one of them needs to be rerendered which creates an overhead. Also this issue might be seen in the browser console as a big warning.Correct piece of code for that would be
`tags.map((t, index) => {
          return <Tag key={index} color={theme.main.colors.first} style={{borderRadius: "8px", height: "16px", fontSize: "10px", lineHeight: "14px"}}>
            {t}
          </Tag>`

Reply:
key prop is added to the tags